### PR TITLE
untracked: Adding 'owner' parameter to functions that define formulas…

### PR DESCRIPTION
**Pull request by oliverbock**
…, including making it obligatory.  Added AddFactbaseProvenance() (experimental).

### CODE REVIEW

File: R/Factbase.R [hunk 1]
```diff
@@ -383,15 +385,18 @@ UpdateFactbasePenetrationFormula <- function(metric_name, token, numerator, deno
...
     
-    url <- paste0("https://factbase.azurewebsites.net/formula?metric=", URLencode(metric_name), "&definition=", URLencode(definition))
-    if (!is.null(hyperlink))
-        url <- paste0(url, '&hyperlink=', hyperlink)
+    url <- paste0(
+        "https://factbase.azurewebsites.net/formula?metric=",
+        URLencode(metric_name),
+        "&definition=", URLencode(definition),
+        "&definition=", URLencode(hyperlink),
+        "&definition=", URLencode(owner))
```
Comment (jrwishart) [inline R/Factbase.R]: Should the two `&definition=` strings be `&hyperlink=` and `&owner=` here?

**Review by jrwishart — APPROVED** [review]
Not going to block merging since I don't know what these functions do in the whole scheme of things, just had a naive comment about some lines that I thought I'd clarify.

File: R/Factbase.R [hunk 2]
```diff
@@ -438,11 +445,49 @@ UpdateFactbaseRatioFormula <- function(metric_name, token, numerator, denominato
...
```
Comment (jrwishart) [inline R/Factbase.R]: Same as above, should `&definition` change to hyperlink and owner?

File: R/Factbase.R (hunk 1 above)
Comment (oliverbock) [reply R/Factbase.R]: Good catch.  Thanks, @jrwishart

**Pull request merged**